docs/core.md

```diff
@@ -313,6 +313,19 @@ app({
   actions: {
     addOne: state => state + 1
   },
-  plugins: [Logger]
+  mixins: [Logger]
 })
 ```
+Mixins can also extend other mixins:
```

Comment:
@dodekeract Only way I could think of is if you intentionally cause a circular reference in the tree of mixins somewhere.  But I don't think that could happen naturally.